app/views/layouts/application.html.slim

```diff
@@ -21,9 +21,8 @@ html lang="#{I18n.locale}" class='no-js'
     <!--[if IE 8]>
       = javascript_include_tag 'es5-shim.min'
     <![endif]-->
-    == javascript_include_tag 'misc/i18n-strings'
-    - unless controller_name == 'home'
-      == javascript_include_tag 'application'
+    == javascript_include_tag 'i18n-strings'
```

Comment:
should this be `javascript_pack_tag`?

Reply:
No, the `i18n-strings` use ERB so I've left it in the app directory for now. One thing to think about after we handle this may be to move that into a template like we do with the session timeout javascript.